features/step_definitions/no_property_address.rb

```diff
@@ -0,0 +1,5 @@
+Given(/^I do not have a property address to provide$/) do
+  @deed = Deed.new(1)
+  @deed.address = ''
+  @deed.address = ''
```

Comment:
Also does this need step need to include the @deed_hash = @deed.to_hash to convert it into a hash ready to send to the API?